Bottom: fb9939a0f9100b292a698d3c24b719095776c9e2
Top:    bb71773f72decca0c522eb1ea891e04541be7b0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 05:29:54 +0100

Refresh of try-debugging-draw-speed

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 8ab100c..5e4510b 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -614,7 +614,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 0
+#if 1
   if (x_prev >= 0)
     {
       draw_crosshair (x_prev, y_prev, z_prev);
@@ -632,7 +632,7 @@ ghid_show_crosshair (gboolean show)
     {
       glBegin (GL_QUADS);
 
-#if 0
+#if 1
       if (x_prev >= 0)
         {
           glVertex3i (0,                  y_prev - VCD,        z_prev);
@@ -1686,12 +1686,10 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
 
   glViewport (0, 0, widget->allocation.width, widget->allocation.height);
 
-#if 1
   glEnable (GL_SCISSOR_TEST);
   glScissor (ev->area.x,
              widget->allocation.height - ev->area.height - ev->area.y,
              ev->area.width, ev->area.height);
-#endif
 
   glMatrixMode (GL_PROJECTION);
   glLoadIdentity ();
